schema.sql

```diff
@@ -36,4 +36,116 @@ ADD FOREIGN KEY (species_id) REFERENCES species(id);
 
 ALTER TABLE animals
 ADD COLUMN owner_id INT,
-ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+--create vets table--
+create table vets (
+    id GENERATED ALWAYS AS IDENTITY PRIMARY KEY, 
+    name VARCHAR(255), 
+    age INT, 
+    date_of_graduation DATE
+);
+
+/*There is a many-to-many relationship between 
+the tables species and vets: a vet can specialize 
+in multiple species, and a species can have multiple 
+vets specialized in it. Create a "join table" called 
+specializations to handle this relationship.*/
+
+create table specializations (species_id INT, vets_id INT);
```

Comment:
* Kindly add all these data below 👇 as it is a[ requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Insert%20the%20following,Jan%2011th%2C%202021.)

> Insert the following data for vets:
> Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
> Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
> Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
> Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
> Insert the following data for specialties:
> Vet William Tatcher is specialized in Pokemon.
> Vet Stephanie Mendez is specialized in Digimon and Pokemon.
> Vet Jack Harkness is specialized in Digimon.
> Insert the following data for visits:
> Agumon visited William Tatcher on May 24th, 2020.
> Agumon visited Stephanie Mendez on Jul 22th, 2020.
> Gabumon visited Jack Harkness on Feb 2nd, 2021.
> Pikachu visited Maisy Smith on Jan 5th, 2020.
> Pikachu visited Maisy Smith on Mar 8th, 2020.
> Pikachu visited Maisy Smith on May 14th, 2020.
> Devimon visited Stephanie Mendez on May 4th, 2021.
> Charmander visited Jack Harkness on Feb 24th, 2021.
> Plantmon visited Maisy Smith on Dec 21st, 2019.
> Plantmon visited William Tatcher on Aug 10th, 2020.
> Plantmon visited Maisy Smith on Apr 7th, 2021.
> Squirtle visited Stephanie Mendez on Sep 29th, 2019.
> Angemon visited Jack Harkness on Oct 3rd, 2020.
> Angemon visited Jack Harkness on Nov 4th, 2020.
> Boarmon visited Maisy Smith on Jan 24th, 2019.
> Boarmon visited Maisy Smith on May 15th, 2019.
> Boarmon visited Maisy Smith on Feb 27th, 2020.
> Boarmon visited Maisy Smith on Aug 3rd, 2020.
> Blossom visited Stephanie Mendez on May 24th, 2020.
> Blossom visited William Tatcher on Jan 11th, 2021.